hbw/app/javascript/packs/hbw/components/form/select_table.js.jsx

```diff
@@ -25,13 +25,31 @@ modulejs.define('HBWFormSelectTable',
         };
       }
 
-      getInputValues = (value) => {
+      initNullablaTable (value) {
         if (this.props.params.multi) {
           return JSON.parse(value) || [];
         } else {
-          return value || '';
+          return value || 'null';
         }
-      };
+      }
+
+      initNotNullablaTable (value) {
+        if (this.props.params.multi) {
+          return JSON.parse(value) || [this.getFirstRowId()];
+        } else {
+          return value || this.getFirstRowId();
+        }
+      }
+
+      initTable (value) {
+        if (this.props.params.nullable) {
+          return this.initNullablaTable(value);
+        } else {
+          return this.initNotNullablaTable(value);
+        }
+      }
+
+      getFirstRowId = () => String(this.getChoices()[0][0])
```

Comment:
Why do you need to cast it to string?

Reply:
Because select element has IDs of type string
I moved translation to  String outside of getFirstRowId function